### PR TITLE
Add metadata to car

### PR DESCRIPTION
**Pull request by praneelc**
Updated metadata.yaml;

 Source Verification: [https://colab.research.google.com/drive/1ELl_3JTJY_W3Pgr3a6k5rvYpxZ4VGm7z#scrollTo=P6tcV-_JUIvo](https://colab.research.google.com/drive/1ELl_3JTJY_W3Pgr3a6k5rvYpxZ4VGm7z#scrollTo=P6tcV-_JUIvo)

### CODE REVIEW

**Comment by praneelc** [comment]
Failed check is showing the same error as the dna.yaml PR;
Permission denied to GitHub to access PMLB repo for 'deployment' step in check

**Comment by trangdata** [comment]
Hi @praneelc: I noticed one typo in the colab notebook:
```
df_source['doors'] = df_source['doors'].replace(df_source['doors'].unique(), df_pmlb['buying'].unique())
```
On the right, I think it should be `'doors'` not `'buying'`.
After fixing, I think we can remove all the checks and simply do:
```
set(df_pmlb.itertuples(index=False)) == set(df_source.itertuples(index=False))
```
and that should return True. I think some rows are permuted.

File: datasets/car/metadata.yaml [hunk 1]
```diff
@@ -1,26 +1,40 @@
 #Generated automatically by pmlb/write_metadata.py
 dataset: car
-description: None yet. See our contributing guide to help us add one.
-source: None yet. See our contributing guide to help us add one.
-publication: None yet. See our contributing guide to help us add one.
+description: >
+  dataset for car evaluation based on six factors;
+  factors considered are buying price, maintenance price, # doors, # people carriable, size of boot, and estimated safety
+source: M. Bohanec, V. Rajkovic. Expert system for decision making. Sistemica 1(1), pp. 145-157, 1990.
```
Comment (trangdata) [inline datasets/car/metadata.yaml]: Can we add the openml link to the source here?

File: datasets/car/metadata.yaml [hunk 2]
```diff
@@ -1,26 +1,40 @@
...
+  description: accceptability of the car
+  code: 0 = acceptable, 1 = good, 2 = unacceptable, 3 = very good
 features: # list of features in the dataset
   - name: buying
     type: categorical
-    description: null # optional but recommended, what the feature measures/indicates, unit
-    code: null # optional, coding information, e.g., Control = 0, Case = 1
-    transform: ~ # optional, any transformation performed on the feature, e.g., log scaled
+    description: buying price of car (vhigh, high, med, low)
+    code: 3 = vhigh, 0 = high, 2 -> med, 1 -> low
```
Comment (trangdata) [inline datasets/car/metadata.yaml]: Can we replace all the `->` with `=`?

**Review by trangdata — COMMENTED** [review]
Looks great! I just have a couple comments.

**Pull request merged**